stracker/customers/models.py

```diff
@@ -0,0 +1,81 @@
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.contrib.auth.models import PermissionsMixin, UserManager
+from django.contrib.auth.validators import UnicodeUsernameValidator
+from django.core.mail import send_mail
+from django.db import models
+
+
+class Customer(AbstractBaseUser, PermissionsMixin):
```

Comment:
Customer ismi mi yoksa user ismi mi daha uygun olur? Hem app için hem de model adı için

Reply:
User base django'da sıkça kullanılan bir kelime olduğu için, bir karışıklık olması ihtimaline karşın customer demeyi tercih ettim.

Konuşmayı istediğim konulardan biri bu. User bir son kullanıcı olarak kulağıma çok daha uygun geliyor. Ancak bir django developer olarak user gibi kelimelerden kaçınmalı mıyız?

Reply:
Kaçınılmasına gerek yok ancak CustomUser olabilir. (Bkz. https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#substituting-a-custom-user-model)